pkg/replica/backup.go

```diff
@@ -301,7 +303,8 @@ func (rb *BackupStatus) CompareSnapshot(snapID, compareSnapID, volumeID string)
 		Offset: -1,
 	}
 
-	if err := preload(&rb.replica.volume); err != nil {
+	rb.replica.volume.initializeSectorLocation(nilFileIndex)
+	if err := rb.replica.volume.preload(); err != nil {
```

Comment:
`if err := rb.replica.Preload(); err != nil {`

Reply:
Thank you. Fixed